Thumbprint/Targets/ThumbprintPublicResources/Assets.xcassets/contentActions/filter--medium.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Not positive, but might need to include a copy of https://github.com/feathericons/feather/blob/master/LICENSE in `ThumbprintPublicResources/Assets.xcassets/` for these icons to be safe since they weren't created by Thumbtack and therefore have their own license (which is also open-source, but different from the license governing the rest of this repo)